docs/using_as_library.md

```diff
@@ -0,0 +1,205 @@
+# Using as a library
+
+The Automation Broker implements the Open Service Broker API. As a broker it
+provides quite a bit of behavior that might want to be used by other broker
+authors, for example, asynchronous job support, ...
+
+## Vendoring
+
+To get the Automation Broker as a dependency, here is an example of the
+`Gopkg.toml` used by `dep`. You can use any of the types supported by the `dep`
```

Comment:
Here is an excerpt from `Gopkg.toml` used by [golang/dep](https://golang.github.io/dep/) that adds the Automation Broker as a dependency.

Reply:
addressed